tests/installation/addon_products_sle.pm

```diff
@@ -43,6 +43,19 @@ sub handle_all_packages_medium {
     # The legacy module is required if upgrade from previous version (bsc#1066338)
     push @addons, 'legacy' if get_var('UPGRADE') && !grep(/^legacy$/, @addons);
 
+    # For SLES12SPx and SLES11SPx to SLES15 migration, follow modules are need at least
+    # SLE-Module-Basesystem15
+    # SLE-Module-Containers15
+    # SLE-Module-Desktop-Applications15
+    # SLE-Module-Legacy15
+    # SLE-Module-Server-Applications15
+    # SLE-Module-Web-Scripting15
+    if (get_var('UPGRADE') && !sle_version_at_least('15', version_variable => 'HDDVERSION')) {
+        my @default_addon = qw/contm desktop legacy serverapp script/;
```

Comment:
The variable name "default_addon" is some bit confusing, here you try to define a list of modules which are required to upgrade SLE 11/12 to 15. Please think about it.